docs/_docs/focus/convert.md

```diff
@@ -57,35 +62,33 @@ The following mapping is assuming you have all amortized cost rows and only comm
 | PublisherName              | PublisherName                                       | None                                                                                                                                                                  |
 | RegionId                   | focus:RegionName                                    | Lowercase and remove spaces                                                                                                                                           |
-| RegionName                 | ResourceLocation                                    | Map using [Regions data file](../../_reporting/data/README.md#-regions)<sup>3</sup>                                                                                   |
+| RegionName                 | ResourceLocation                                    | Map ResourceLocation to RegionName using [Regions data file](../../_reporting/data/README.md#-regions)<sup>2</sup>                                                    |
 | ResourceId                 | ResourceId                                          | None                                                                                                                                                                  |
 | ResourceName               | ResourceName                                        | None                                                                                                                                                                  |
-| ResourceType               | ResourceType                                        | Map using [Resource types data file](../../_reporting/data/README.md#-resource-types)                                                                                 |
-| ServiceCategory            | ResourceType                                        | Map using [Services data file](../../_reporting/data/README.md#-services)                                                                                             |
-| ServiceName                | ResourceType                                        | Map using [Services data file](../../_reporting/data/README.md#-services)                                                                                             |
+| ResourceType               | SingularDisplayName (lookup)                        | Map ResourceType to SingularDisplayName using [Resource types data file](../../_reporting/data/README.md#-resource-types)                                             |
+| ServiceCategory            | ServiceCategory (lookup)                            | Map ResourceType to SerivceCategory using [Services data file](../../_reporting/data/README.md#-services)                                                             |
```

Comment:
I believe for the `Services` reference data file, we need to map based on `ConsumedService` and `ResourceType`, based on this documentation - https://microsoft.github.io/finops-toolkit/data#%EF%B8%8F-services.  It's not unique just by `ResourceType`.

Reply:
Yes and no. I need to go look at the updated mapping we use internally, but there are only 1 or 2 services that need a ConsumedService mapping. The rest are all ResourceType. I have a task on my backlog to merge Services and ResourceTypes and just call out the exceptions separately. Any thoughts on how you'd prefer to see that here?

Reply:
No, thoughts exactly.  I processed the `Services` reference file locally and found duplicate values with just `ResourceType`, but when I combined it with `ConsumedService` it was unique.  As long as it is unique on the lookup, probably not much to document.

Reply:
```suggestion
| ServiceCategory            | ServiceCategory (lookup)                            | Map ConsumedService and ResourceType to SerivceCategory using [Services data file](../../_reporting/data/README.md#-services)                                                             |
```